### DISCUSSION TITLE
Hi there, I've been farming for a few days.  I will see in my wallet still 0 ?

### DISCUSSION THREAD

**Discussion by sagrolikar** [Chia-Network/chia-blockchain#3615] [General]
Hi there, I've been farming for a few days.  I will see in my wallet still 0 ? my plots is 700gb 7plots farming so how can earn xch ,  i am new in this pls guid me what is next step .

**Comment by RexLAz** [comment]
Keep ploting you will get more chances to win the block

**Comment by nomadengineer** [comment]
At this point you need AT LEAST 10 TB of plots to stand any chance in 1 month!

**Reply by Christoffer2020** [reply]
What probability per TB per month? Is there a formula for calculating this?

**Reply by RexLAz** [reply]
It depends on how many plots you can do per day, then you will know how many plots you can do in a month. The large ssd you have and hdd you got, the more plots you will have.

**Reply by nomadengineer** [reply]
use https://chiacalculator.com/

**Reply by nomadengineer** [reply]
https://chialinks.com/
another very good resource
